### PR TITLE
Godeps: Bump rkt api, go grpc, go protobuf.

### PR DESCRIPTION
**Pull request by jonboulle**
Supersedes #19931 - this is that PR rebased and with Godeps licenses fixed.

### CODE REVIEW

**Comment by vishh** [comment]
ok to test

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -150,123 +146,123 @@
...
-			"Comment": "v2.2.2-4-ge0c7768",
-			"Rev": "e0c7768f94cdc268b2fce31ada1dea823f11f505"
+			"Comment": "v2.2.5",
+			"Rev": "bc9ddf260115d2680191c46977ae72b837785472"
 		},
 		{
 			"ImportPath": "github.com/coreos/etcd/discovery",
-			"Comment": "v2.2.2-4-ge0c7768",
-			"Rev": "e0c7768f94cdc268b2fce31ada1dea823f11f505"
+			"Comment": "v2.2.5",
```
Comment (vishh) [inline Godeps/Godeps.json]: nit: I wish `"github.com/coreos/etcd/client"` was self-sufficient without needing a lot of other packages.

**Comment by lavalamp** [comment]
Can you fix the commit? It appears as though someone else made it.

**Comment by jonboulle** [comment]
What do you mean, fix the commit? Can @yifan-gu just ack this somehow? (We went through something similar in https://github.com/kubernetes/kubernetes/pull/20553)

**Comment by yujuhong** [comment]
@jonboulle is not the original author (@yifan-gu is). I'm not aware of any method to manually override the cla bot without consent from the commit author, @lavalamp, maybe you know?

**Comment by lavalamp** [comment]
OK. Yeah, if @yifan-gu could ack this (just say something here) that'd be great.

**Comment by yifan-gu** [comment]
ack

**Comment by yifan-gu** [comment]
LGTM

**Comment by mikedanese** [comment]
cc @wojtek-t @gmarek w.r.t. scalability. we are bumping the etcd client version here.

ping @googlebot

**Comment by mikedanese** [comment]
LGTM

**Comment by gmarek** [comment]
@mikedanese - OK, we'll monitor test results.

**Comment by jonboulle** [comment]
anything we need to do here to kick the cla/google check?

**Comment by gmarek** [comment]
There's a human-approved label, so don't worry about it.

**Pull request merged**